packages/manager/apps/pci-instances/src/components/cart/Cart.tsx

```diff
@@ -7,17 +7,21 @@ import { TInstanceCreationForm } from '@/pages/instances/create/CreateInstance.p
 export const Cart = () => {
   const { control } = useFormContext<TInstanceCreationForm>();
   const instanceName = useWatch({ control, name: 'name' });
+  const instanceQuantity = String(
+    useWatch({ control, name: 'quantity', defaultValue: 1 }),
```

Comment:
I think we can get all properties like `const { name: instanceName, quantity: instanceQuantity }`with an name array param, no?